src/executor.rs

```diff
@@ -149,55 +150,71 @@ impl Executor {
 
     /// See `std::process::Command::spawn`
     pub fn spawn(&mut self) -> Result<ExecutorChild> {
+        self.log_command();
         let result = match self {
-            Executor::Wet(c) => {
+            Executor::Wet(c) | Executor::Damp(c) => {
                 debug!("Running {:?}", c);
                 // We should use `spawn()` here rather than `spawn_checked()` since
                 // their semantics and behaviors are different.
                 #[allow(clippy::disallowed_methods)]
                 c.spawn().map(ExecutorChild::Wet)?
             }
-            Executor::Dry(c) => {
-                c.dry_run();
-                ExecutorChild::Dry
-            }
+            Executor::Dry(_) => ExecutorChild::Dry,
         };
 
         Ok(result)
     }
 
     /// See `std::process::Command::output`
     pub fn output(&mut self) -> Result<ExecutorOutput> {
+        self.log_command();
         match self {
-            Executor::Wet(c) => {
+            Executor::Wet(c) | Executor::Damp(c) => {
                 // We should use `output()` here rather than `output_checked()` since
                 // their semantics and behaviors are different.
                 #[allow(clippy::disallowed_methods)]
                 Ok(ExecutorOutput::Wet(c.output()?))
             }
-            Executor::Dry(c) => {
-                c.dry_run();
-                Ok(ExecutorOutput::Dry)
-            }
+            Executor::Dry(_) => Ok(ExecutorOutput::Dry),
         }
     }
 
     /// An extension of `status_checked` that allows you to set a sequence of codes
     /// that can indicate success of a script
     #[allow(dead_code)]
     pub fn status_checked_with_codes(&mut self, codes: &[i32]) -> Result<()> {
+        self.log_command();
         match self {
-            Executor::Wet(c) => c.status_checked_with(|status| {
+            Executor::Wet(c) | Executor::Damp(c) => c.status_checked_with(|status| {
                 if status.success() || status.code().as_ref().is_some_and(|c| codes.contains(c)) {
                     Ok(())
                 } else {
                     Err(())
                 }
             }),
-            Executor::Dry(c) => {
-                c.dry_run();
-                Ok(())
+            Executor::Dry(_) => Ok(()),
+        }
+    }
+
+    fn log_command(&self) {
+        match self {
+            Executor::Wet(_) => (),
+            Executor::Damp(c) => {
+                log_command(
+                    "Executing {program_name} {arguments}",
```

Comment:
```suggestion
                    "Executing: {program_name} {arguments}",
```
Doesn't match the key in `app.yml`, results in "Executing {program_name} {arguments}" messages.